pkgs/top-level/all-packages.nix

```diff
@@ -30874,15 +30874,15 @@ with pkgs;
 
   bleachbit = callPackage ../applications/misc/bleachbit { };
 
-  blender = callPackage  ../applications/misc/blender {
-    # LLVM 11 crashes when compiling GHOST_SystemCocoa.mm
-    stdenv = if stdenv.isDarwin then llvmPackages_10.stdenv else stdenv;
-    inherit (darwin.apple_sdk.frameworks) Cocoa CoreGraphics ForceFeedback OpenAL OpenGL;
-  };
+  # Backwards compatibiility
+  blender = blenderPackages.latest; 
+  blender-hip = blenderPackages-hip.latest;
 
-  blender-with-packages = callPackage ../applications/misc/blender/wrapper.nix { };
+  blenderPackages = callPackage ../applications/misc/blender { };
```

Comment:
blenderPackages is not the attribute naming scheme for versioning. Look at how it's done in e.g. `boost`.

Reply:
Moved to `blender3_x`, `blender3_x-hip`, and with `blender`, `blender-lts`, `blender-hip` etc

Reply:
Please see
https://nixos.org/manual/nixpkgs/stable/#sec-versioning for file organization
and last bullet of https://nixos.org/manual/nixpkgs/stable/#sec-package-naming for blender_3

Reply:
So `blender_3_x`, `blender-hip_3_x`?

Reply:
The `_x` is omitted.

Reply:
> The `_x` is omitted.

I don't understand what you mean.

The example in the link you gave me (https://nixos.org/manual/nixpkgs/stable/#sec-package-naming) with their example has `json-c_0_9` and `json-c_0_11`.

If you mean `blender_3x`, personally I think that's confusing, given previously blender has versions like `2.93`, and blender uses these to indicate API breaking changes (major and minor versions).